### PR TITLE
Docs update for integration to geocat-comp

### PR DESCRIPTION
**Pull request by erogluorhan**
Updates repo-specific docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,17 +7,42 @@ contribution guidelines:
 
 # Adding new functions to GeoCAT-f2py repo
 
+Adding a new function to GeoCAT-f2py may almost always be due to the need for replicating a NCL function
+under GeoCAT-comp.
```
Comment (anissa111) [inline CONTRIBUTING.md:11]: I'm not sure what this means.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -65,29 +90,47 @@ should be added to the following file:
...
+10. Implementing a wrapper function would, most of time, be straightforward. A Python wrapper function 
+(i.e. user API function) for each NCL function would handle input data preprocessing (sanity checks, auto-chunking, 
+etc.) and then would make use of a second (i.e. internal) wrapper function that handles tasks such as missing 
+value handling and Fortran function calls for every single Dask chunk (where Dask paralelization is applicable). 
+For example, see 
 
-   `$GEOCAT-F2PY/src/geocat/f2py/_init_.py`
+    `$GEOCAT-F2PY/src/geocat/f2py/rcm2rgrid_wrapper.py`.
+   
+    In cases where no Dask parallelization is needed, even a single wrapper function would be enough.
```
Comment (anissa111) [inline CONTRIBUTING.md:119]:
```suggestion
    In cases where no build in Dask parallelization is needed, even a single wrapper function would be enough. 
```

**Review by anissa111 — COMMENTED** [review]
I just have a couple in line comments, this looks good. Thank you for getting this together!

File: CONTRIBUTING.md (hunk 1 above)
Comment (erogluorhan) [reply CONTRIBUTING.md]: Agreed and rephrased.

File: CONTRIBUTING.md (hunk 2 above)
Comment (erogluorhan) [reply CONTRIBUTING.md]: Addressed

**Review by anissa111 — APPROVED** [review]
This looks great!

**Review by michaelavs — APPROVED** [review]
Looks good!

**Pull request merged**